src/components/Person/Person.jsx

```diff
@@ -1 +1,21 @@
-// export const Person = ({ person }) => ();
+const genderTitle = {
+  m: 'wife',
+  f: 'husband',
+};
+
+export const Person = ({ person }) => {
+  const { name, age, partnerName, isMarried, sex } = person;
+  const genderLine = `${partnerName} is my ${genderTitle[sex]}`;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age && <p className="Person__age">{`I am ${age}`}</p>}
```

Comment:
```suggestion
      {!!age && <p className="Person__age">{`I am ${age}`}</p>}
```
Highly important if the age is 0.
In this case, we render 0, because it will be a fallback from && operator.